charts/internal/shoot-cert-management-seed/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cert-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "cert-management.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - name: metrics
```

Comment:
We also need the `networking.gardener.cloud/from-prometheus: allowed` label for the cert-management pod in order to scrape the metrics.

Reply:
I double checked it and we already have this label, sorry about the confusion. When I tested the PR the scraping target was down, though. Let me find the reason and come back.

Reply:
Resolving this as it seemed to be a transient problem in the setup.